lib/jekyll/filters.rb

```diff
@@ -68,22 +68,33 @@ def slugify(input, mode = nil)
     end
 
     # Format a date in short format e.g. "27 Jan 2011".
+    # It allows to use ordinal format as well, in both UK
```

Comment:
This comment would read better if it was written thusly:
```
Ordinal format is also supported, in both the UK and US formats.
```

Please keep the example dates in this comment when you make the language change. I left them out of the above sentence just to keep that shorter.